components/EpisodeVideoDescriptionPanel.brs

```diff
@@ -0,0 +1,466 @@
+sub init()
+    m.top.panelSize = "full"
+    m.top.isFullScreen = true
+    m.top.leftPosition = 130
+    m.top.focusable = true
+    m.top.hasNextPanel = false
+
+    m.top.updateFocus = false
+    m.top.observeField("updateFocus", updateFocus)
+    m.top.observeField("start", "showVideoDetails")
+    m.top.isVideo = false
+end sub
+
+sub error()
+    print "EpisodeVideoDescriptionPanel:error()"
+    source = "EpisodeVideoDescriptionPanel:"
+    errorMessage = m.global.utilities.callFunc("GetErrorMessage", {errorCode: m.readItemTask.error, source: source})
+    print errorMessage
+    font  = CreateObject("roSGNode", "Font")
+    font.uri = "pkg:/fonts/NotoSans-Regular-w1251-rename.ttf"
+    font.size = 24
+
+    m.dialog = createObject("roSGNode", "Dialog")
+    m.dialog.title = recode("Ошибка")
+    m.dialog.titleFont = font
+    m.dialog.message = recode(errorMessage)
+    m.dialog.messageFont = font
+    m.top.dialog = m.dialog
+end sub
+
+sub showVideoDetails()
+    serial  = m.top.serial
+    season  = m.top.season
+    episode = m.top.episode
+
+    loadSettings()
+
+    availableWidth  = m.top.width / 2 - 120
+    availableHeight = m.top.height - 100
+
+    widthHeight = availableWidth * 250 / 165
+    heightWidth = availableHeight * 165 / 250
+
+    if widthHeight <= availableHeight
+        width  = availableWidth
+        height = widthHeight
+    else
+        height = availableHeight
+        width  = heightWidth
+    end if
+
+    left = availableWidth / 2 - width / 2
+
+    imageUri = episode.thumbnail
+
+    poster = createObject("roSGNode", "Poster")
+    poster.translation = [left, 0]
+    poster.width = width
+    poster.height = height
+    poster.loadDisplayMode = "scaleToFit"
+    poster.uri = imageUri
+    m.top.appendChild(poster)
+
+    m.font24  = CreateObject("roSGNode", "Font")
+    m.font24.uri = "pkg:/fonts/NotoSans-Regular-w1251-rename.ttf"
+    m.font24.size = 24
+
+    m.font18  = CreateObject("roSGNode", "Font")
+    m.font18.uri = "pkg:/fonts/NotoSans-Regular-w1251-rename.ttf"
+    m.font18.size = 18
+
+    m.font16  = CreateObject("roSGNode", "Font")
+    m.font16.uri = "pkg:/fonts/NotoSans-Regular-w1251-rename.ttf"
+    m.font16.size = 16
+
+    textLeft = left + width + 50
+
+    'HACKHACK: the unusedSpace here is a total banana. There is unused space on the screen which doesn't belong
+    'to the panel and is not accounted in m.top.width. I couldn't figure out how to calculate it so hack.
+    unusedSpace = 135
```

Comment:
Is it the same for all TV sizes?

Reply:
I have no idea, I just copied this logic from `VideoDescriptionPanel.brs`